scanpipe/pipes/d2d.py

```diff
@@ -269,7 +269,9 @@ def scan_for_java_package(location, with_threading=True):
     return scancode._scan_resource(location, scanners, with_threading=with_threading)
 
 
-def save_java_package_scan_results(codebase_resource, scan_results, scan_errors):
+def save_java_package_scan_results(
+    codebase_resource, scan_results, scan_errors, status
```

Comment:
Why this change?

Reply:
fixed, we're using a separate `save_func` for legal files and no longer need to pass the status to the `save_func`.